### ISSUE TITLE
failed to apply new package git pull tag

### ISSUE THREAD

**Issue by ExpandingMan** [JuliaRegistries/Registrator.jl#352]
Not really a big deal, but for some reason I don't understand the registrator bot failed to apply the new package tag [here](https://github.com/JuliaRegistries/General/pull/45387).  Don't know why, the pulls right next to it that did get the tag don't really look any different to me.

**Comment by DilumAluthge** [comment]
Yeah, this is a known issue. Basically, Registrator.jl has two code paths:
1. GitHub comment-bot
2. JuliaHub web UI

The comment-bot code path includes the logic for adding labels, but the web UI code path does not.

**Comment by DilumAluthge** [comment]
@ericphanson This means that new package registrations created from the JuliaHub web UI won't show up in the Slack or Zulip feeds.

**Comment by ExpandingMan** [comment]
> @ericphanson This means that new package registrations created from the JuliaHub web UI won't show up in the Slack or Zulip feeds.

This was what drew my attention to it.
